Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN npm prune --production
 FROM node:16-bullseye-slim
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
-ENV CACHE_BREAKER ${GIT_REF}
+ENV GIT_REF ${GIT_REF:-xxxxxxxxxxxxxxxxxxx}
```

Comment:
env variables don't need a default - see https://github.com/ministryofjustice/hmpps-template-typescript/pull/233/files
also better to use new `ENV MY_VAR=my_value` syntax - see _Alternative syntax_ section in https://docs.docker.com/engine/reference/builder/#env

Reply:
Ha, I didn't have one but included it because it was in the template!